src/core_modules/capture-core/components/WidgetEnrollmentEventNew/Validated/getConvertedAddEvent.js

```diff
@@ -44,6 +46,8 @@ export const getAddEventEnrollmentServerData = ({
                 orgUnit: orgUnitId,
                 trackedEntityInstance: teiId,
                 enrollment: enrollmentId,
+                dueDate: mainDataServerValues.eventDate,
+                orgUnitName: organisationUnits[orgUnitId].name,
```

Comment:
I think I have removed every dependency to data that are being set in the app sync for the new enrollment event page. So, if we can get the organisation unit name from here instead it would be great: https://github.com/dhis2/capture-app/blob/7daf0e63d0fcbb23278dc0af0c4111db057cb603/src/core_modules/capture-core/components/WidgetEnrollmentEventNew/Validated/Validated.container.js#L31

(yes, we should try to remove the app sync soon for this page 🙂 )